seafile/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: seafile
+category: Files
```

Hey @IMPranshu , due to a recent change the category values should be lowercase. 

```suggestion
category: files
```